tests/integration/test_storage_kerberized_kafka/kerberos_image_config.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
+
+
+[[ "TRACE" ]] && set -x
+
+: ${REALM:=TEST.CLICKHOUSE.TECH}
```

Comment:
There are few tiny issues there, which https://www.shellcheck.net/ can show (nothing very important, but would be nice to fix)